### ISSUE TITLE
Unable to run "python manager.py start"

### ISSUE THREAD

**Issue by CaptainNicodemus** [swar/Swar-Chia-Plot-Manager#158]
**Followed the installation prossess on windows power shell and cmd admin, when I get to running "python manager.py start" i get "psutil.AccessDenied: psutil.AccessDenied (pid=2396)"**

 (venv) C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main>python manager.py start
Traceback (most recent call last):
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\_common.py", line 447, in wrapper
    ret = self._cache[fun]
AttributeError: _cache

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\_pswindows.py", line 679, in wrapper
    return fun(self, *args, **kwargs)
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\_common.py", line 450, in wrapper
    return fun(self)
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\_pswindows.py", line 766, in exe
    exe = cext.proc_exe(self.pid)
PermissionError: [WinError 24] The program issued a command but the command length is incorrect: '(originated from NtQuerySystemInformation)'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manager.py", line 29, in <module>
    start_manager()
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\commands.py", line 20, in start_manager
    if get_manager_processes():
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\processes.py", line 30, in get_manager_processes
    if not re.search(r'^pythonw?(?:\d+\.\d+|\d+)?(?:\.exe)?$', process.name(), flags=re.I):
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\__init__.py", line 617, in name
    name = self._proc.name()
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\_pswindows.py", line 750, in name
    return os.path.basename(self.exe())
  File "C:\Users\nicod\Documents\Swar-Chia-Plot-Manager-main\venv\lib\site-packages\psutil\_pswindows.py", line 681, in wrapper
    raise convert_oserror(err, pid=self.pid, name=self._name)
psutil.AccessDenied: psutil.AccessDenied (pid=2396)

**Comment by acoleman-apc** [comment]
you need to run the . ./activate (from the chia-blockchain dir I think) before trying to start the manager or chia

**Comment by zachnintendo123** [comment]
I'm getting the same error on Windows 10 Pro.  If the GUI is already open, is the . ./activate still needed?  I think it's a psutil version error, but I am not sure.

**Comment by mutoo185** [comment]
How can I fix it?thanks!

**Comment by zachnintendo123** [comment]
I managed to fix it.  I posted this to my reddit account as well:

I am using Python 3.9.5. Under “This PC>Properties>Advanced System Properties>Environment Variables”, I created a new System Variable called “PYTHON_HOME” that is a pathway to my Python39 folder that was installed. It should be under AppData\Local\Programs\Python\Python39.

Then under System Variables still, click Edit, click New, and add %PYTHON_HOME%. Click New again and add this and replace your user name: C:\Users\USERNAME\AppData\Local\Programs\Python\Python39\Scripts

I updated to Chia 1.1.5 due to the connection issues earlier today on 1.1.4. Then make sure you config.ymal file is configured properly.

I would delete any other versions of Python under the Python folder from above that are present on your system. Python 3.8 was causing the error for me. After that, I could start the manager with powershell and it’s plotting correctly now.
